docs/settings/storagenetwork/manual.md

```diff
@@ -0,0 +1,311 @@
+---
+sidebar_position: 1
+sidebar_label: Manual Storage Network Configuration
+title: ""
+---
+
+# Manual Storage Network Configuration 
+
+This page describe the maunal method to configure Longhorn Storage Network in Harvester. 
+
+## Prerequisite
+
+1. Make sure that Harvester version is v1.1.0 or above.
+2. Prepare phyical interfaces networks for the storage network, and make sure the VLAN configuration in the external switch side.
```

Comment:
```suggestion
2. Prepare physical interface networks for the storage network, and ensure the VLAN configuration on the external switch side is correct.
```

Please use tools to scan the rest of the document and see if there are any obvious spelling issues.

Reply:
fix